### PR TITLE
[CA-1989] Add step up to sdk ui

### PR DESCRIPTION
**Pull request by sachaMemmir5**
https://reach5.atlassian.net/browse/CA-1989

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import socialLoginWidget from './widgets/socialLogin/socialLoginWidget';
 import webAuthnWidget from './widgets/webAuthn/webAuthnDevicesWidget';
 import { logError } from './helpers/logger';
 import mfaCredentialsWidget from "./widgets/mfa/MfaCredentialsWidget";
+import mfaStepUpWidget from "./widgets/stepUp/mfaStepUpWidget";
```
Comment (axuan25) [inline src/client.js:15]: lint

File: src/widgets/stepUp/mfaStepUpWidget.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {createMultiViewWidget} from "../../components/widget/widget";
+import {deepDefaults} from "../../helpers/deepDefaults";
+import {createForm} from "../../components/form/formComponent";
+import radioboxField from "../../components/form/fields/radioboxField";
+import {Info, Intro} from "../../components/miscComponent";
+import {simpleField} from "../../components/form/fields/simpleField";
+import {toQueryString} from "../../helpers/queryString";
```
Comment (axuan25) [inline src/widgets/stepUp/mfaStepUpWidget.jsx:8]: (single quotes)

**Review by axuan25 — APPROVED** [review]

File: src/widgets/stepUp/mfaStepUpWidget.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            ({...resp, ...factor})
+        )
+    }
+
+    render() {
+        const { amr, showIntro, i18n } = this.props
+        return <div>
+            {showIntro && <Intro>{i18n('mfa.select.factor')}</Intro>}
+            <StartPasswordlessForm
+                options={amr.map(factor => ({ key: factor, value: factor, label: factor}))}
```
Comment (nbrr) [inline src/widgets/stepUp/mfaStepUpWidget.jsx:69]: Is that preselected when there only is one factor available?

**Review by nbrr — APPROVED** [review]

File: src/widgets/stepUp/mfaStepUpWidget.jsx (hunk 3 above)
Comment (sachaMemmir5) [reply src/widgets/stepUp/mfaStepUpWidget.jsx:69]: You are right I changed the behaviour to preselect the only factor available when there is only one second factor.

**Pull request merged**